src/Views/ProgressView.vala

```diff
@@ -183,6 +183,20 @@ public class ProgressView : AbstractInstallerView {
             value = 0
         };
 
+        // Sectors may also be constructed using different units of measurements, such as
+        // by megabytes and percents. The library author can choose whichever unit makes
+        // more sense for their use cases.
```

Comment:
I don't really understand the reason of this comment in this code, it sounds like Distinst documentation